### ISSUE TITLE
[Suggestion] Scrolling preserves the mouse position

### ISSUE THREAD

**Issue by mlv** [Querz/mcaselector#209]
I suggest a change to how zooming works, such that when you zoom in or out, the point where the mouse currently is doesn't move, rather than the middle of the display. That way you could zoom out to see the whole map, move the mouse to a remote base, then zoom in and see that base, without having to drag that base to the middle of the display. It would make for a natural and smooth way to explore the map.

**Comment by Querz** [comment]
I agree that this is useful, and it was just a minor change. Added in https://github.com/Querz/mcaselector/commit/1de1101dbb287841ee1fc1a775d4ad59b1a3cadf and will be in the next release (most likely tomorrow).

**Comment by FeedTheChunk** [comment]
> 
> 
> I agree that this is useful, and it was just a minor change. Added in [1de1101](https://github.com/Querz/mcaselector/commit/1de1101dbb287841ee1fc1a775d4ad59b1a3cadf) and will be in the next release (most likely tomorrow).

Thank you!  I'm really looking forward to this.
